### PR TITLE
Fixed typos on the page for "if" and "sand"

### PR DESCRIPTION
**Pull request by vijayysisodia**
Updated this page and fixed typos from
a) "To take advantage if this solution" to "To take advantage of this solution"
b) "easy to access sand share content" to "easy to access and share content"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/home-site-plan.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ After you've created your home site, it’s time to plan the launch and make sur
...
+| ![image of a globe and monitor](media/icon-launch-viva.png)   | **Integrate the home site into Microsoft Teams using Viva Connections** | Expand the reach of the home site and help meet users where they're already working by making it easy to access and share content all in one place. After you have a home site and the global navigation enabled and customized in the SharePoint app bar, you’ve met the requirements to [integrate the home site into Microsoft Teams](/SharePoint/viva-connections) using Viva Connections.Learn more about the [Viva Connections end-user experience](https://support.microsoft.com/office/your-intranet-is-now-in-microsoft-teams-8b4e7f76-f305-49a9-b6d2-09378476f95b).|
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/home-site-plan.md:160]:
```suggestion
| ![image of a globe and monitor](media/icon-launch-viva.png)   | **Integrate the home site into Microsoft Teams using Viva Connections** | Expand the reach of the home site and help meet users where they're already working by making it easy to access and share content all in one place. After you have a home site and the global navigation enabled and customized in the SharePoint app bar, you’ve met the requirements to [integrate the home site into Microsoft Teams](/SharePoint/viva-connections) using Viva Connections. Learn more about the [Viva Connections end-user experience](https://support.microsoft.com/office/your-intranet-is-now-in-microsoft-teams-8b4e7f76-f305-49a9-b6d2-09378476f95b).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @vijayysisodia, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Holland-ODSP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by Holland-ODSP — APPROVED** [review]

**Comment by scanum** [comment]
Hi @Holland-ODSP - Could you kindly merge this PR? Thanks

**Comment by Holland-ODSP** [comment]
Hey Manuela,

I'd be happy to merge but I don't see that as an option in Github. This is what I see:

***@***.***

**Pull request merged**